Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -1447,88 +977,68 @@ final class KSRAnalyticsTests: TestCase {
   }
 
   func testTrackGotoCreatorDetailsClicked() {
-    let dataLakeClient = MockTrackingClient()
     let segmentClient = MockTrackingClient()
-    let ksrAnalytics = KSRAnalytics(dataLakeClient: dataLakeClient, segmentClient: segmentClient)
+    let ksrAnalytics = KSRAnalytics(segmentClient: segmentClient)
 
     ksrAnalytics.trackGotoCreatorDetailsClicked(
       project: .template
     )
 
-    XCTAssertEqual(["CTA Clicked"], dataLakeClient.events)
     XCTAssertEqual(["CTA Clicked"], segmentClient.events)
-
-    XCTAssertEqual("creator_details", dataLakeClient.properties.last?["context_cta"] as? String)
     XCTAssertEqual("creator_details", segmentClient.properties.last?["context_cta"] as? String)
-
-    XCTAssertEqual("project", dataLakeClient.properties.last?["context_page"] as? String)
     XCTAssertEqual("project", segmentClient.properties.last?["context_page"] as? String)
-
-    self.assertProjectProperties(dataLakeClient.properties.last)
     self.assertProjectProperties(segmentClient.properties.last)
   }
 
   func testTrackPledgeCTAButtonClicked_FixState() {
-    let dataLakeClient = MockTrackingClient()
     let segmentClient = MockTrackingClient()
     let project = Project.template
     let loggedInUser = User.template |> \.id .~ 42
 
     let ksrAnalytics = KSRAnalytics(
-      dataLakeClient: dataLakeClient,
       loggedInUser: loggedInUser,
       segmentClient: segmentClient
     )
 
     ksrAnalytics.trackPledgeCTAButtonClicked(stateType: .fix, project: project)
 
-    XCTAssertEqual(["Manage Pledge Button Clicked"], dataLakeClient.events)
-    XCTAssertEqual(["Manage Pledge Button Clicked"], segmentClient.events)
+    XCTAssertEqual([], segmentClient.events)
```

Comment:
We should confirm this. @tobitech

Reply:
It's actually not useful anymore we're only handling `.pledge` case for this method `trackPledgeCTAButtonClicked()` inside `KSRAnalytics.swift` so that `.fix` state doesn't do anything